test/grouping.jl

```diff
@@ -898,7 +898,11 @@ end
     @test gd[1] isa SubDataFrame
     @test gd[1] == view(df, [1, 5], :)
     @test_throws BoundsError gd[5]
-    @test_throws ArgumentError gd[true]
+    if VERSION < v"1.0.0-"
```

Comment:
Better turn around the test so that the first test behavior is the one for Julia 1.0 and later (as elsewhere).